src/decisionengine/framework/dataspace/dataspace.py

```diff
@@ -81,25 +83,23 @@ def __init__(self, config):
         :type config: :obj:`dict`
         :arg config: Configuration dictionary
         """
-
-        self.logger = logging.getLogger()
-        self.logger.debug('Initializing a dataspace')
+        logger.debug('Initializing a dataspace', msg='')
 
         # Validate configuration
         if not config.get('dataspace'):
-            self.logger.exception("Error in initializing DataSpace!")
+            logger.exception('Error in initializing DataSpace!')
             raise DataSpaceConfigurationError('Configuration is missing dataspace information: dataspace key not found.')
         elif not isinstance(config.get('dataspace'), dict):
-            self.logger.exception("Error in initializing DataSpace!")
+            logger.exception('Error in initializing DataSpace!')
             raise DataSpaceConfigurationError('Invalid dataspace configuration: '
                                               'dataspace key must correspond to a dictionary')
         try:
             self._db_driver_name = config['dataspace']['datasource']['name']
             self._db_driver_module = config['dataspace']['datasource']['module']
             self._db_driver_config = config['dataspace']['datasource']['config']
         except KeyError:
-            self.logger.exception("Error in initializing DataSpace!")
-            raise DataSpaceConfigurationError('Invalid dataspace configuration')
+            logger.exception('Invalid dataspace configuration!')
+            raise DataSpaceConfigurationError()
```

Comment:
Shouldn't the exception still contain the error message?

Reply:
Yes ,it should. I have added that back in.